servercfg/serverconf.go

```diff
@@ -427,7 +428,16 @@ func GetPublicIP() (string, error) {
 	var err error
 
 	iplist := []string{"https://ip.server.gravitl.com", "https://ifconfig.me", "https://api.ipify.org", "https://ipinfo.io/ip"}
+	publicIpService := os.Getenv("PUBLIC_IP_SERVICE")
+	if publicIpService != "" {
+		fmt.Println("User provided public IP service is", publicIpService)
+
+		// prepend the user-specified service so it's checked first
+		iplist = append([]string{publicIpService}, iplist...)
+	}
+
 	for _, ipserver := range iplist {
+		fmt.Println("Running public IP check with service", ipserver)
```

Comment:
logger.Log()

Reply:
Thanks Matt, would like to. We're working on how to get that done.